### DISCUSSION TITLE
GD32F103 processor?

### DISCUSSION THREAD

**Discussion by Trilis29** [bassamanator/Sovol-SV06-firmware#21] [Q&A]
I can't connect to printer, tried everything. Maybe this is related to GD32 processor?

**Comment by bassamanator** [comment]
> I can't connect to printer, tried everything. Maybe this is related to GD32 processor?

This has nothing to do with the processor.

Please read the Before You Begin section.

**Reply by Trilis29** [reply]
Which logs and config do you need? Maybe I am missing something.
